### PR TITLE
[18.0][MIG] document_page_procedure: Migration to 18.0

### PR DESCRIPTION
**Pull request by natuan9**
Depend on:
- https://github.com/OCA/knowledge/pull/504

### CODE REVIEW

File: document_page_procedure/readme/CONFIGURE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+To be able configure your template of procedures, you need to install
+mgmtsystem_manual:
+
+- go to Management Systems \> Configuration \> Categories
+- select the Procedure category
+- customize the template
```
Comment (ajaniszewska-dev) [inline document_page_procedure/readme/CONFIGURE.md]: imo this not really clear/outdated, looking at the runboat:
either we go to mgmt system -> configuration -> settings and we need to tick (i believe by default when module is installed):
image

or what is meant here to go to mgmt system -> documentation -> pick a Procedure template and edit it?
image

Comment (max3903) [inline document_page_procedure/readme/CONFIGURE.md]:
```suggestion
To configure your template of procedures:

- Go to Management Systems \> Configuration \> Categories
- Select the Procedure category
- Customize the content
```

File: document_page_procedure/readme/CREDITS.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+The development of this module has been financially supported by:
```
Comment (max3903) [inline document_page_procedure/readme/CREDITS.md]:
```suggestion
The migration of this module has been financially supported by:
```

File: document_page_procedure/readme/DESCRIPTION.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+This module provides a Procedure Template. If mgmtsystem_manual will be
+installed you can change initial template.
```
Comment (max3903) [inline document_page_procedure/readme/DESCRIPTION.md]:
```suggestion
This module provides a Procedure template.

- Go to Management Systems \> Documentation \> Procedures
- Create a new Procedure
- Fill in the title and the content
```

File: document_page_procedure/readme/CONFIGURE.md (hunk 1 above)
Comment (natuan9) [reply document_page_procedure/readme/CONFIGURE.md]: If the mgmtsystem_manual module is not installed, we need to go to Knowledge > Pages > Categories instead.

Comment (natuan9) [reply document_page_procedure/readme/CONFIGURE.md]: > i believe by default when module is installed

That's correct, and when you disable this field, the Document Management - Wiki - Procedures module will also be uninstalled.

**Review by max3903 — APPROVED** [review]

**Comment by max3903** [comment]
@natuan9 Can you squash the last 3 commits?

**Pull request closed without merging**